MAKE/Makefile.sisu_gcc

```diff
@@ -84,4 +84,4 @@ INC_PROFILE = -I$(LIBRARY_PREFIX)/mpich2/$(MPT_VERSION)/$(CC_BRAND)/$(CC_BRAND_V
 INC_EIGEN = -I$(LIBRARY_PREFIX)/eigen/
 INC_DCCRG = -I$(LIBRARY_PREFIX)/dccrg_new_neighbours/
 INC_VECTORCLASS = -I$(LIBRARY_PREFIX)/vectorclass
-INC_FSGRID = -I$(LIBRARY_PREFIX)/fsgrid
+INC_FSGRID = -I$(HOME)/fsgrid
```

Comment:
I guess this shouldn't be part of the overall commit here?

Reply:
Until you fix the fsgrid version on sisu, as said.